src/Controller/CreationController.php

```diff
@@ -38,6 +38,11 @@ public function addCreation()
                 $errors[] = 'Veuillez ajouter un prix';
             }
 
+            if (preg_match('/\D+/', $_POST['price'])!=0)
```

Comment:
bien joué la regex, mais pour le coup tu ferais mieux d'utilier is_int() http://php.net/manual/en/function.is-int.php